packages/orchestration/src/orchestration-util.ts

```diff
@@ -0,0 +1,57 @@
+import type {
+  AzureContentSafety,
+  DataRepositoryType,
+  DocumentGroundingFilter,
+  GroundingModuleConfig,
+  InputFilteringConfig,
+  OutputFilteringConfig
+} from './client/api/schema/index.js';
+
+/**
+ * Convenience function to create Azure content filters.
+ * @param filter - Filtering configuration for Azure filter. If skipped, the default Azure content filter configuration is used.
+ * @returns An object with the Azure filtering configuration.
+ */
+export function buildAzureContentFilter(
+  filter?: AzureContentSafety
+): InputFilteringConfig | OutputFilteringConfig {
+  if (filter && !Object.keys(filter).length) {
+    throw new Error('Filter property cannot be an empty object');
+  }
+  return {
+    filters: [
+      {
+        type: 'azure_content_safety',
+        ...(filter && { config: filter })
+      }
+    ]
+  };
+}
+
+export type DocumentGroundingFilterWrapper = Pick<DocumentGroundingFilter,
+  'id' | 'search_config' | 'data_repositories' | 'data_repository_metadata' | 'document_metadata' | 'chunk_metadata'> & {
```

Comment:
@jjtang1985 Do you think making `data_repositories` optional would make sense? If user doesn't set it, we set it to * 
```
"data_repositories": ["*"],
```

Reply:
[pp] For me personally, if `["*"]`, then this parameter can be optional.